gateway-ha/src/main/java/io/trino/gateway/ha/config/MonitorConfiguration.java

```diff
@@ -21,6 +22,10 @@ public class MonitorConfiguration
 
     private int retries;
 
+    private Duration queryTimeout = Duration.valueOf("10s");
```

Comment:
Use `new Duration(10, SECONDS)` instead. Parsing flow is redundant when we already know the unit.